src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/ResponsibilitiesLogicTests.kt

```diff
@@ -0,0 +1,83 @@
+package org.vaccineimpact.api.tests.logic
+
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.api.app.controllers.helpers.ResponsibilityPath
+import org.vaccineimpact.api.app.errors.UnknownObjectError
+import org.vaccineimpact.api.app.logic.RepositoriesResponsibilitiesLogic
+import org.vaccineimpact.api.app.repositories.ModellingGroupRepository
+import org.vaccineimpact.api.app.repositories.ScenarioRepository
+import org.vaccineimpact.api.app.repositories.SimpleDataSet
+import org.vaccineimpact.api.app.repositories.TouchstoneRepository
+import org.vaccineimpact.api.models.TouchstoneStatus
+import org.vaccineimpact.api.models.TouchstoneVersion
+import org.vaccineimpact.api.test_helpers.MontaguTests
+
+class ResponsibilitiesLogicTests : MontaguTests()
+{
+    private val groupId = "group-1"
+    private val touchstoneVersionId = "touchstone-1"
+    private val scenarioId = "scenario-1"
+
+    @Test
+    fun `can validate Responsibility Path`()
```

Comment:
These have been moved from the BurdenEstimateLogicTests class - the logic is unchanged